OrbitQt/orbittreeview.h

```diff
@@ -14,8 +14,9 @@ class OrbitTreeView : public QTreeView {
   Q_OBJECT
  public:
   explicit OrbitTreeView(QWidget* parent = nullptr);
-  void Initialize(DataViewType a_Type);
-  void SetDataModel(std::shared_ptr<DataView> a_Model);
+  void Initialize(DataView* data_view, bool enable_extended_selection,
+                  bool set_fixed_font);
```

Comment:
What about two `enum class` here instead of the bools? That would also make the parameter-comments in orbitmainwindow.cpp superfluous.